scripts/ssl-manager.js

```diff
@@ -490,6 +501,33 @@ function SSLManager(config) {
         return me.getFileUrl("scripts/" + scriptName);
     };
 
+    me.initCustomConfigs = function() {
+        var CUSTOM_CONFIG = nodeManager.getCustomSettingsPath(),
+            configs = [],
+            body,
+            resp,
+            tmp;
+
+        resp = nodeManager.readFile(CUSTOM_CONFIG, config.nodeGroup);
+
+        if (resp.result == Response.FILE_PATH_NOT_EXIST) return {
+            result: 0
+        }
+        if (resp.result != 0) return resp;
+
+        body = resp.body.replace(/\n$/, '');
+        configs = body.split('\n');
+
+        for (var i = 0, n = configs.length; i < n; i++) {
+            tmp = configs[i].split('=');
+            config[tmp[0]] = me.initBoolValue(tmp[1].replace(/\"/g, ""));
```

Comment:
no needs use function **initBoolValue** there because custom settings can be different types. Webroot bool value defines [there](https://github.com/dzotic9/lets-encrypt/blob/master/scripts/ssl-manager.js#L101)